site/content/docs/api-checks/_index.md

```diff
@@ -13,64 +13,108 @@ aliases:
+Setup scripts allow you to do last minute processing of test data and request options. These scripts execute before any requests are made. Teardown scripts are run after the HTTP request has finished, but before any assertions are validated. With a maximum execution time of 10 seconds, these scripts are useful for things like signing HMAC requests, requesting tokens, setting up or cleaning up test data and scrubbing sensitive response data for regulatory reasons.
 
 [Read more about setup and teardown scripts](setup-teardown-scripts/)
-![setup and teardown scripts](/docs/images/api-checks/overview-scripts.png)
 
-**4. Response time limits**
+### Response time limits
 
-Our [time limits](limits) are customisable, sometimes API's can be slow, but not broken. We call that **degraded**. We let you specify when an API check should be marked as **degraded** and when it should be marked as **failed**.
-![response times](/docs/images/api-checks/overview-response-time.png)
+Sometimes APIs can be slow, but not broken. We call this **degraded**. You can set [response time limits](/docs/api-checks/limits/) to specify when an API check should be marked as **degraded** and when it should be marked as **failed**.
 
-**5. Assertions**
+### Assertions
 
-This is where you determine whether the response of the HTTP request is correct or within acceptable bounds.
+This is where you determine whether the response of the HTTP request is correct or not.
 You can assert on different sources. These could be:
 - The HTTP status code returned from the API
 - Something missing or required within the response body
 - A specific response header
 - A specific response time
 
 [Read more about assertions](assertions)
-![assertions](/docs/images/api-checks/overview-assertions.png)
 
-**6. Locations**
+### Scheduling & locations
+
+You can configure your checks to run from our [public](/docs/monitoring/global-locations/) locations, or use a Checkly Agent to host your own [private](/docs/private-locations/) locations. If you don't select more than one location and you've disabled retrying checks from the same location, we will pick a random location when retrying checks.
+
+Checkly runs your API checks based on an interval you set. The shortest interval you can run is every 10 seconds and the longest is every 24 hours.
+
+### Retries & alerting
+
+Select your preferred [retry strategy](/docs/alerting-and-retries/retries/) for failed checks.
+
+Our alerting is pretty flexible. You can configure any of our [alert channels](/docs/alerting-and-retries/alert-channels/) for whichever checks you like. If we don't have your preferred alert method, why not try out our [webhooks](/docs/alerting-and-retries/webhooks/)?
+> [SMS](/docs/alerting-and-retries/sms-delivery/) and [phone call](/docs/alerting-and-retries/phone-calls/) alerts are only available on our [Team and Enterprise plans](https://www.checklyhq.com/pricing) 
```

Comment:
I would not put this callout here. There are more alerting channels that are on different plans, and it will be a nightmare to keep these types of callouts up to date. People will bump into these plan limits naturally as they use the product.